modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,159 @@
+= Use Studio to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To use Anypoint Studio to configure a connector:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+Follow these best practices when you configure an input source and connector in Studio:
+
+*  Configure an associated global element for the input source and connector.
+* Create a `.yaml` file or `.properties` file to contain your properties, and then reference the file from the configuration.
+* Configure a reconnection strategy for the input source and target connector.
+
+For information about these best practices, see <<introduction/intro-connector-configuration-overview.adoc#best-practices,Best Practices for Configuring Connectors>>.
+
+== Install the Connector
+
+Installing the connector adds it to a Studio project. Studio provides two ways to install a connector:
+
+* Use the Exchange application in the Studio taskbar.
+
+* Use the Exchange icon in the Mule Palette view.
+
+(*Reviewers, is there any advantage to using one of these methods for installing a connector* )
```

Comment:
I can see only visual differences between the two.
Also it is possible to install the connector by manually adding a dependency into the `pom.xml`, in fact that's what the other two options do in the background.

Reply:
I am going to cover the pom.xml piece in a separate topic.

Reply:
I will be covering the pom.xml in a separate topic.